src/config/index.ts

```diff
@@ -153,6 +153,7 @@ export const ENTITY_CHANGE_EMAIL = ENTITY_URL + EMAIL;
 export const ENTITY_CHANGE_LEGAL_FORM = ENTITY_URL + LEGAL_FORM;
 export const ENTITY_CHANGE_GOVERNING_LAW = ENTITY_URL + LAW_GOVERNED;
 export const ENTITY_CHANGE_PUBLIC_REGISTER = ENTITY_URL + "#is_on_register_in_country_formed_in";
+export const CONCAT_VALUE_SEPARATOR = ",";
```

Comment:
Nice. I would just suggest expanding the name a bit - CONCATENATED_VALUES_SEPARATOR.

Reply:
Ok, updated this.